src/module.ts

```diff
@@ -19,10 +19,13 @@ export default defineNuxtModule<ModuleOptions>({
   setup (options, nuxt) {
     const resolver = createResolver(import.meta.url)
 
+    nuxt.options.runtimeConfig.medusa = defu(nuxt.options.runtimeConfig.medusa, {
```

Comment:
I think here you need to name it private to not be accessible by the frontend.

Reply:
only `runtimeConfig.public` is exposed to the frontend Because is **explicitly name ** public :)

Reply:
> I think here you need to name it private to not be accessible by the frontend.

@Baroshem I was thinking that `medusa` would be duplicated and didn't know which name should I use.
But `private` would be better. I will update soon.

> only `runtimeConfig.public` is exposed to the frontend Because is **explicitly named** public :)

@pi0 That I was unsure about that. As far as I know from docs, `public` would be the special name for exposing.
Thank you for your information.

Reply:
080d29ab12f2d9d673a4720f044d26d464f347af